io/serialport/25-serial.js

```diff
@@ -358,7 +358,13 @@ module.exports = function(RED) {
                                 // do the timer thing
                                 if (spliton === "time") {
                                     // start the timeout at the first character
-                                    if (!obj.tout) {
+                                    if ( i == 1 ) {
```

Comment:
This mode should only be enabled when in split on time from last character mode - not the current timeout mode.

Reply:
Hmm... keep in mind this version does not include the "new" behavior.
It behaves exactly as before: it starts the timeout the at the first character. It just uses a different test to establish it's the first character, as obj.tout could have been set for a different purpose already.
Anyway, I'm about to push the commit for the new behavior.